.babelrc

```diff
@@ -1,10 +1,26 @@
 {
-  "presets": [["@babel/preset-env", { "modules": false }]],
-  "plugins": ["@babel/plugin-syntax-dynamic-import"],
+  "presets": [
+    ["@babel/preset-env", {
+      "modules": false
+    }]
+  ],
   "env": {
     "test": {
       "presets": ["@babel/preset-env"],
-      "plugins": ["dynamic-import-node"]
+      "plugins": ["dynamic-import-node", "@babel/plugin-syntax-dynamic-import"]
+    },
+    "esm": {
+      "presets": [
+        [
+          "@babel/preset-env",
```

Comment:
Any value to changing this to https://github.com/babel/preset-modules/blob/master/README.md instead?

Reply:
Ah good idea I’ll try that out

Reply:
Weird - I tried it out but it **increased** the bundle size, instead of decreasing it.

base branch - 23778
with preset-env - 20571
with preset-modules - 23769

I don't know if I'm missing a configuration or not? But looks like preset-env is superior afaict

Reply:
Weird. Did you try loose:true as well?

Reply:
Just tried `loose`, no improvement. I'm going to try to identify which 300 bytes are being removed from the preset-env version

Reply:
Found it! Using `yarn build:analyze` I discovered that this file is being included when using preset-modules, but is not included when using preset-env:

```txt
file:            rollupPluginBabelHelpers.js
bundle space:    9.52 %
rendered size:   4.777 KB
original size:   42.897 KB
code reduction:  88.86 %
dependents:      5
  - /src/applications/timeouts.js
  - /src/parcels/mount-parcel.js
  - /src/lifecycles/prop.helpers.js
  - /src/lifecycles/load.js
  - /src/applications/apps.js
```

This is likely due to [this rollup-plugin-babel behavior](https://github.com/rollup/rollup-plugin-babel#helpers). It apparently works with preset-env, but not with preset-modules?? We could probably open up an issue on rollup-plugin-babel to see what they say about it.

Reply:
Wait actually I was wrong on this. That babel helpers file is included in both output bundles. Still trying to diagnose this 😢

Reply:
preset-env analysis: https://pastebin.com/MNH8DLgq
preset-modules analysis: https://pastebin.com/3Ezj9VPm

Reply:
After much research, this appears to be related to the babel `_typeof` helper https://github.com/babel/babel/blob/70c0ed512a5fb2e8f461c7d14e493a3ce131e686/packages/babel-helpers/src/helpers.js#L14

Also see https://github.com/babel/babel/commit/e1839e5ec9c10ffadac51aa58fbc1e6753c6b27f#diff-ffd431f9489b95ec613d71c2ade19e4dR17

It looks like some code inside of preset-modules is triggering the symbol typeof check unnecessarily, which is increasing the bundle size. Probably just need to wait for a newly published version of preset-modules that has the fix.